src/components/Collapse.jsx

```diff
@@ -53,7 +53,11 @@ class Collapse extends Component {
       this.setExpanded();
     }
 
-    if (this.props.onRest && e.target === this.content && e.propertyName === 'height') {
+    if (
+      this.props.onRest &&
```

Comment:
We should use destructuring her: 
```
const { onRest } = this.props
```

The latest version of eslint should report this.....

Reply:
✅